### PR TITLE
fix: article template errors

### PR DESCRIPTION
**Pull request by tuncaulubilge**
There are couple of red screens on the app when we enable all the templates, this PR fixes these

- Unknown template types were causing a crash. Now they fallback to articleMainStandard (indepth were causing an issue)
- Articles with empty content were causing a crash, such as the cartoon article
- First paragraph can be empty or could have a markup, which were causing a crash

### CODE REVIEW

File: packages/article/src/article.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const Article = props => {
   const { article } = props;
   const { template = "mainstandard" } = article || {};
 
-  const Component = templates[template];
+  const Component = templates[template] || ArticleMainStandard;
```
Comment (aronshamash) [inline packages/article/src/article.js]: `const Component = templates[template || 'mainstandard']` then can remove the default value of template on line 16.

File: packages/article-skeleton/src/article-skeleton.js [hunk 2]
```diff
@@ -127,7 +127,9 @@ class ArticleSkeleton extends Component {
     }
 
     const newContent = [...dataSource.content];
-    newContent[0] = insertDropcapIntoAST(newContent[0], template);
+    if (newContent && newContent.length > 0) {
```
Comment (nehasri89) [inline packages/article-skeleton/src/article-skeleton.js]: should we do the same check for web too while we are here

Comment (tuncaulubilge) [reply packages/article-skeleton/src/article-skeleton.js]: Good point, will do :+1:
this is especially a problem on the cartoon article, I'll update the pr

**Comment by nehasri89** [comment]
should have some tests for empty para and the use cases where it crashed

**Comment by tuncaulubilge** [comment]
Added few more tests @nehasri89 @colincclark

**Pull request merged**